app/models/event.rb

```diff
@@ -38,6 +37,10 @@ def self.upcoming
     where(published: true).order('started_at').last
   end
 
+  def self.current
+    find_by('started_at >= :beginning_of_day AND finished_at < :end_of_day', beginning_of_day: Time.zone.now.beginning_of_day, end_of_day: Time.now.end_of_day)
```

Comment:
why wee need this `current` workaround? Could we just use ID of event? `HomeController` already has detection login of event that must be displayed at main page.